Tests/Whiteboard_14_MaxStack/maxTest.java

```diff
@@ -0,0 +1,20 @@
+package Whiteboard_14_MaxStack;
+
+import org.junit.jupiter.api.Test;
+
+import java.util.Stack;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class maxTest {
+
+    @Test
+    void test1() {
```

Comment:
The data type for `ss` should be MaxStack, not `Stack`, since `MaxStack` should be the new class that extends `Stack`.  The `max()` method will only available on `MaxStack` objects.